Show address on Wallet page

alexliebowitz opened the pull request:
- Remembers address from previous views using local storage
 - Also improved layout and added help text

kauffj commented:
1) This has the same problem as @sonatagreen 's solution: we need to re-verify they have access to a wallet address stored in local storage (e.g. I deleted by `.lbryum` file or I ran with `--wallet lbrycrd` flag.)

2) Need to do a better job understanding what others are working on, this is basically exactly what Sonata already did.

alexliebowitz commented:
I sure do :/ I now remember seeing Sonata's PR in one of the GitHub email updates, but I think it was already closed by the next time I checked out the current PRs. Sorry, Sonata!

Anyway, this one does also have some significant design improvements, plus the help text, and it also includes an Address component so we can use standard styles for addresses throughout the UI.

I just added some commits that do what you asked for (refresh the address if it doesn't match the current wallet). With Jack's help, I added functionality to the daemon to pass along the needed info from from Lbrycrd and Lbryum. Here's the PR for that: https://github.com/lbryio/lbry/pull/149

kauffj commented:
:+1: this was well done but we need to figure out better styling solution
